project/Settings.scala

```diff
@@ -56,56 +56,80 @@ object Settings {
     val awsSdk = "1.11.89"
     val awsCommons = "0.12.2"
     val csvCommons = "1.4"
-    val pprint = "0.5.4"
+    val pprint = "0.5.6"
     val scalaCheck = "1.13.4"
     val akkaPersistenceInMemory = "2.5.15.2"
     val akkaPersistenceJdbc = "3.5.0"
-    val bluebus = "0.3.1-DRT"
+    val bluebus = "0.3.2-DRT"
     val postgres = "42.2.2"
     val sshJ = "0.24.0"
     val jodaTime = "2.9.4"
     val playJsonJoda = "2.6.9"
     val exposeLoader = "0.7.1"
     val log4Javascript = "1.4.15"
     val typesafeConfig = "1.3.0"
-    val reactHandsontable = "0.3.1"
+    val reactHandsontable = "3.1.2"
     val spark = "2.4.3"
     val pac4jSaml = "2.0.0-RC1"
     val openSaml = "2.6.1"
     val drtBirminghamSchema = "1.2.0"
     val drtCirium = "48"
-    val drtLib = "58"
+    val drtLib = "59"
```

Comment:
looks like this is a branch build of drtlib. do we need to PR that branch and then update the build number here?

Reply:
thanks! I forgot to add a PR for this.

Reply:
i think the drtlib version should be bumped to 61 for the latest master build